### ISSUE TITLE
VirtScreen does not run on ArchLinux

### ISSUE THREAD

**Issue by vs0uz4** [kbumsik/VirtScreen#15]
After installing it with:
$ yaourt virtscreen

When executing the command:
$ virtscreen

The error reported on the screen is as follows
```
Traceback (most recent call last):
  File "/usr/bin/virtscreen", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3105, in <module>
    @_call_aside
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3089, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3118, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 578, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 895, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 781, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'PyQt5>=5.10.1' distribution was not found and is required by virtscreen
```

Checking the pyqt5 versions in ArchLinux, I noticed that the installed version is higher than the one requested.
```
extra / pyqt5-common 5.11.2-2 [installed]
    Common PyQt files shared between python-pyqt5 and python2-pyqt5
extra / python-pyqt5 5.11.2-2 [installed]
    A set of Python 3.x bindings for the Qt5 toolkit
```

**Comment by kbumsik** [comment]
Hello,
I just noticed that. this happens because [`python-pyqt5` Arch Linux package does not have metadata that can be recognized while checking that dependencies](https://bugs.archlinux.org/task/58887). I am working on it now. I will let you know when it's finished :)

**Comment by kbumsik** [comment]
Fixed in f62fa66. Thanks for reporting!

**Comment by kbumsik** [comment]
Closing the issue cuz I believe it does not happen anymore. Thanks for reporting!
